examples/use-assertions/test.js

```diff
@@ -0,0 +1,32 @@
+import { Selector } from 'testcafe';
+
+fixture `Use build-in assertions`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+test('Using snapshot API shorthands', async t => {
+    const triedTestCafeCheckbox = Selector('#tried-test-cafe');
+
+    await t
+        .expect(triedTestCafeCheckbox.checked).notOk()
+        .click(triedTestCafeCheckbox)
+        .expect(triedTestCafeCheckbox.checked).ok();
+
+    var slider        = Selector('.ui-slider-handle');
+    var sliderTick    = Selector('.slider-value');
+    var initialOffset = await slider.offsetLeft;
+
+    await t
+        .dragToElement(slider, sliderTick.withText('9'))
+        .expect(slider.offsetLeft).gte(initialOffset);
+});
+
+test('Using DOM node snapshot', async t => {
+    const developerNameInput = Selector('#developer-name');
+    const initialValue       = await developerNameInput().value;
+
+    await t
+        .expect(initialValue).eql('', 'input is empty')
+        .typeText(developerNameInput, 'Peter Parker');
+
+    await t.expect(developerNameInput().value).contains('Parker', 'input has text');
```

Comment:
I think we should show directly how to use a snapshot in assertions. The common case: use get a snapshot a check several properties, like:
```js
var inputSnapshot = await Selector(...);

t
.expect(inputSnapshot.prop1).eql(value1)
.expect(inputSnapshot.prop2).notEq(value2);
```